doc/getting-started/getting-started.rst

```diff
@@ -469,6 +444,14 @@ To generate a scenario configuration file and launch scripts:
       .. image:: ./images/gsg_config_launch_load.png
          :class: drop-shadow
 
+      Of the six User VMs, we will use User VM 3 and modify its default settings to run Ubuntu 20.04.
+
+   #. Scroll down, find **User VM 3**, and change the **mem_size** to **1024**.
+      Ubuntu 20.04 needs at least 1024 megabytes of memory to boot.
+
+      .. image:: ./images/gsg_config_mem.png
+         :class: drop-shadow
+
```

Comment:
While we're here we should also change where the User VM 3 image is found (instead of manually editing the launch script later in these instructions).   A bit further down the screen there's an entry in the virtio_devices section called "block" that says "./YaaG.img" we should change this value to "/home/acrn/acrn-work/ubuntu-20.04.4-desktop-amd64.iso".